### DISCUSSION TITLE
the number of org packages gives 0

### DISCUSSION THREAD

**Discussion by njzjz** [lowlighter/metrics#998] [Q&A] [answered]
See:

https://github.com/deepmodeling/.github/blob/main/github-metrics.svg
https://github.com/orgs/deepmodeling/packages

It should give 6, but actually gives 0.

**Comment by lowlighter** [comment, marked as answer]
This will be fixed by #1005, but you'll need to provide the additional `read:packages` scope unfortunately.

The GraphQL API doesn't expose (currently?) packages published on ghcr.io (see related [github community discussion](https://github.community/t/github-container-registry-ghcr-io-packages-not-appearing-in-api/130133))
As a workaround, the REST API provides [`listPackagesForOrganization`/`listPackagesForUser`](https://octokit.github.io/rest.js/v18#packages-list-packages-for-organization) endpoints, but these requires the mentioned scopes to work
